resources/keywords/import_provider_account.resource

```diff
@@ -146,8 +146,6 @@ User Enters Data To Import RDS Provider Account
     Input Text    ${txtBxRdsAwsAccsKeyID}    ${RDS.awsAccessKeyId}
     Input Text    ${txtBoxRdsAwsSecKeyID}    ${RDS.awsSecretAccessKeyId}
     Input Text    ${txtBoxRdsAwsRegion}    ${RDS.awsRegion}
-    Input Text    ${txtBoxAckResrceTags}    ${RESOURCE_TAGS_EMPTY}
```

Comment:
I hope these variables should be there and we have to add these into test-variables.yaml file to handle the values

Reply:
Yes of course.  We will add them when we will add the relevant test case.